validator/testcases/regex.py

```diff
@@ -450,6 +451,33 @@ def tests(self):
         )
 
 
+@register_generator
+class Gecko50RegexTests(CompatRegexTestHelper):
+    """Regex tests for Firefox 50 updates."""
+
+    VERSION = FX50_DEFINITION
+
+    def tests(self):
+        instances = ('draggesture', 'dragdrop', 'ondraggesture', 'ondragdrop')
+
+        msg = (
+            'The draggesture and dragdrop events are no longer supported. '
+            'You should use the standard equivalents instead: '
+            'dragstart and drop.')
+
+        description = (
+            msg + ' See %s for more information.'
+            % MDN_DOC % 'Web/API/HTML_Drag_and_Drop_API')
+
+        yield self.get_test(
+            r'[\'"]?(%s)([\'"]|=)' % ('|'.join(instances)),
+            msg,
+            description,
+            log_function=self.err.warning,
+            compat_type='error'
+        )
```

Comment:
Because `validator.testcases.reges:run_regex_tests` supports a `applicatble` callable which is being set by `CompatRegexTestHelper` which we inherit here that set's it to basically check for `VERSION` being set on the class so that it skips tests if they're not for the particular app version.